src/DependencyInjection/Xsd2PhpExtension.php

```diff
@@ -20,7 +21,12 @@ public function load(array $configs, ContainerBuilder $container)
             $config = array_merge($config, $subConfig);
         }
 
-        $definition = $container->getDefinition('goetas_webservices.xsd2php.naming_convention.' . $config['naming_strategy']);
+        $namingStrategy = $config['naming_strategy'];
+        if (in_array($namingStrategy, ['short', 'long'], true)) {
+            $definition = $container->getDefinition('goetas_webservices.xsd2php.naming_convention.' . $namingStrategy);
+        } else {
+            $definition = new Definition($namingStrategy);
```

Comment:
what about using an alias? so is possible to pass parameters to the naming strategy constructor?